Filters/exceptions.txt

```diff
@@ -1,3 +1,5 @@
+! https://twitter.com/Red_Frame_P02/status/1706931925779366188
+@@||ad2.trafficgate.net^|
```

Comment:
Maybe we should block only `srv2.trafficgate.net`?

Reply:
Ah, likely.